jdaviz/configs/default/plugins/model_fitting/model_fitting.py

```diff
@@ -203,7 +205,8 @@ def user_api(self):
                    'equation', 'equation_components',
                    'add_results', 'residuals_calculate', 'residuals']
         expose += ['calculate_fit', 'clear_table', 'export_table',
-                   'fitted_models', 'get_models', 'get_model_parameters']
+                   'fitted_models', 'get_models', 'get_model_parameters',
+                   'parallel_n_cpu']
```

Comment:
let's holdoff on exposing for now.  And tests will likely need to be updated from passing the kwarg to setting `plg.parallel_n_cpu`